registry/Scripts/script.sh

```diff
@@ -3,6 +3,7 @@
 ERR_APT_INSTALL_TIMEOUT=9           # Timeout installing required apt packages
 ERR_MISSING_CRT_FILE=10             # Bad cert thumbprint OR pfx not in key vault OR template misconfigured VM secrets section
 ERR_MISSING_KEY_FILE=11             # Bad cert thumbprint OR pfx not in key vault OR template misconfigured VM secrets section
+ERR_MISSING_USER_CREDENTIAL=12      # Missing any user credential to connect to server
```

Comment:
`No user credentials were provided` or similar may be better

Reply:
As disscussed kepping it as it is to aling with pre-req.ps1.

Reply:
Sorry, `credentials` needs an S at the end

Reply:
My initial comment was about the comment (`Missing any user credential to connect to server`)